examples/julia-machine-learning/MNIST-example.ipynb

```diff
@@ -0,0 +1,480 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4f705426-0538-4858-9681-f60159298780",
+   "metadata": {},
+   "source": [
+    "# Train a Simple Machine Learning Model Using Julia"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55af3953-a4ac-421d-8fa0-5ff3cffb27ea",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-03-07T14:49:27.748000Z",
+     "iopub.status.busy": "2022-03-07T14:49:27.237000Z",
+     "iopub.status.idle": "2022-03-07T14:49:28.718000Z",
+     "shell.execute_reply": "2022-03-07T14:49:28.718000Z"
+    }
+   },
+   "source": [
+    "![Julia Logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/julia-logo.png \"doc-image\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a1daeeb-2625-406d-a2f8-ebd3e9038864",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This example describes how to run a simple machine learning workflow using the famous MNIST dataset of handwritten digits. We will be labeling images of numbers. \n",
+    "\n",
+    "## Modeling Process\n",
+    "\n",
+    "### Imports\n",
+    "This exercise uses [Flux](https://fluxml.ai/Flux.jl/stable/) to train a simple neural network. "
```

Comment:
Does this seem correct? Also is Images a base function? I don't know enough Julia...

```suggestion
    "This exercise uses [Flux](https://fluxml.ai/Flux.jl/stable/) to train a simple neural network. The code loads several functions from Flux as well as some base functions"
```